hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/datanodes/decommissionSummary.tsx

```diff
@@ -0,0 +1,121 @@
+    this.setState({
+      isLoading: true
+    });
+    if (this.state.record && this.state.summaryData !== 'null' && this.state.summaryData !== 'undefined') {
+      this.fetchDecommissionSummary(this.state.uuid);
+    }
+  }
+
+  fetchDecommissionSummary = async (selectedUuid: any) => {
+    try {
+      const datanodeEndpoint = `/api/v1/datanodes/decommission/info/datanode?uuid=${selectedUuid}`;
+      let infoDatanodeResponse = await axios.get(datanodeEndpoint);
+      let DatanodesDecommissionInfo = [];
+      DatanodesDecommissionInfo = infoDatanodeResponse && infoDatanodeResponse.data && infoDatanodeResponse.data.DatanodesDecommissionInfo[0];
+      this.setState({
+        loading: false,
+        summaryData: DatanodesDecommissionInfo
+      });
+    }
+    catch (error) {
+      this.setState({
+        loading: false,
+        summaryData: []
+      });
+      showDataFetchError(error.toString());
+    }
+  };
+
+  render() {
+    const { summaryData, uuid } = this.state;
+    let content;
+    
+    if ( summaryData && summaryData.length !== 0 && summaryData !== null && summaryData !== undefined && summaryData.datanodeDetails) {   
+      const { datanodeDetails, containers, metrics } = summaryData;
+      content = (
+        <Descriptions size="small" bordered column={1} title={`Decommission Status: DECOMMISSIONING`}>
+          <Descriptions.Item label="Datanode"> <b>{datanodeDetails.uuid}</b></Descriptions.Item>
+          <Descriptions.Item label="Location">({datanodeDetails.networkLocation}/{datanodeDetails.ipAddress}/{datanodeDetails.hostname})</Descriptions.Item>
+          {metrics !== null && metrics !== undefined && Object.keys(metrics).length !== 0 &&
+            <>
+              {<Descriptions.Item label="Decommissioning Started at">{metrics.decommissionStartTime}</Descriptions.Item>}
+              {<Descriptions.Item label="No. of Unclosed Pipelines">{metrics.numOfUnclosedPipelines}</Descriptions.Item>}
+              {<Descriptions.Item label="No. of Unclosed Containers">{metrics.numOfUnclosedContainers}</Descriptions.Item>}
+              {<Descriptions.Item label="No. of Under-Replicated Containers">{metrics.numOfUnderReplicatedContainers}</Descriptions.Item>}
+            </>
+          }
+          {
+            containers !== null && containers !== undefined && Object.keys(containers).length !== 0 &&
+            <>
+              {containers.UnderReplicated && containers.UnderReplicated.length > 0 && <Descriptions.Item label="Under-Replicated">{containers.UnderReplicated}</Descriptions.Item>}
+              {containers.UnClosed && containers.UnClosed.length > 0 && <Descriptions.Item label="Unclosed">{containers.UnClosed}</Descriptions.Item>}
+            </>
+          }
+        </Descriptions>
+      );
+    }
+    // Need to check summarydata is not empty 
+    return (
+      <>
+        { (summaryData !== 'null' && summaryData !== 'undefined' && summaryData && summaryData.length !== 0) ?
```

Comment:
```suggestion
        { (summaryData && summaryData.length !== 0) ?
```

Reply:
Done the changes in latest commit https://github.com/apache/ozone/commit/3e08fb7b48adea2a78e0f77408427da00db453bf and done testing with local cluster data.